test/test-thread-priority.c

```diff
@@ -0,0 +1,109 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
I think that unless this file is substantially based on an existing file with the same copyright header that this could (should?) be 

```suggestion
/* Copyright libuv contributors. All rights reserved.
```

(Further discussion in https://github.com/libuv/libuv/issues/4096.)

Reply:
copyright header updated